src/main/scala/com/microsoft/ml/spark/cognitive/TextAnalyticsSDK.scala

```diff
@@ -18,20 +17,23 @@ import org.apache.spark.sql.functions.col
 import org.apache.spark.sql.types.{DataTypes, StructType}
 import org.apache.spark.sql.{DataFrame, Dataset, Row}
 import com.azure.ai.textanalytics.models
+import org.apache.spark.sql.Encoder
 
 import java.net.URI
 import scala.collection.JavaConverters._
 
 abstract class TextAnalyticsSDKBase[T](val textAnalyticsOptions: Option[TextAnalyticsRequestOptions] = None)
   extends Transformer
   with HasInputCol with HasErrorCol
-  with HasEndpoint with HasSubscriptionKey
+  with HasEndpoint with HasSubscriptionKey with HasOutputCol

```

Comment:
nit: can you move this to be after `HasInputCol`?

Reply:
Corrected in new PR.